yoti_python_sdk/anchor.py

```diff
@@ -1,141 +1,178 @@
-                parsed_anchors.append(Anchor(
-                    anchor_type,
-                    anc.sub_type,
-                    Anchor.decode_asn1_value(extension_value.value),
-                    anc.signed_time_stamp,
-                    crypto_cert))
-
-        return parsed_anchors
-
-    @staticmethod
-    def decode_asn1_value(value_to_decode):
-        extension_value_asn1 = value_to_decode
-        decoder = asn1.Decoder()
-
-        decoder.start(extension_value_asn1)
-        tag, once_decoded_value = decoder.read()
-
-        decoder.start(once_decoded_value)
-        tag, twice_decoded_value = decoder.read()
-
-        utf8_value = twice_decoded_value.decode('utf-8')
-        return utf8_value
-
-    @property
-    def anchor_type(self):
-        return self.__anchor_type
-
-    @property
-    def value(self):
-        return self.__value
-
-    @property
-    def sub_type(self):
-        return self.__sub_type
-
-    @property
-    def signed_timestamp(self):
-        if self.__signed_timestamp is None:
-            return None
-
-        signed_timestamp_object = compubapi.SignedTimestamp()
-        signed_timestamp_object.MergeFromString(self.__signed_timestamp)
-
-        try:
-            signed_timestamp_parsed = datetime.datetime.fromtimestamp(
-                signed_timestamp_object.timestamp / float(1000000))
-        except OSError:
-            print("Unable to parse timestamp from integer: '{0}'".format(signed_timestamp_object.timestamp))
-            return ""
-
-        return signed_timestamp_parsed
-
-    @property
-    def origin_server_certs(self):
-        return self.__origin_server_certs
+import datetime
+import logging
+
+import OpenSSL
+import asn1
+import yoti_python_sdk.protobuf.common_public_api.SignedTimestamp_pb2 as compubapi
+from OpenSSL import crypto
+
+from yoti_python_sdk import config
+
+UNKNOWN_EXTENSION = ""
+SOURCE_EXTENSION = "1.3.6.1.4.1.47127.1.1.1"
+VERIFIER_EXTENSION = "1.3.6.1.4.1.47127.1.1.2"
+
+
+class Anchor:
+    def __init__(
+        self,
+        anchor_type="Unknown",
+        sub_type="",
+        value="",
+        signed_timestamp=None,
+        origin_server_certs=None,
+    ):
+        self.__anchor_type = anchor_type
+        self.__sub_type = sub_type
+        self.__value = value
+        self.__signed_timestamp = signed_timestamp
+        self.__origin_server_certs = origin_server_certs
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        self.idx += 1
+        try:
+            return self.data[self.idx - 1]
+        except IndexError:
+            self.idx = 0
+            raise StopIteration
+
+    next = __next__  # python2.x compatibility.
+
+    @staticmethod
+    def parse_anchors(anchors):
+        if anchors is None:
+            return None
+
+        parsed_anchors = []
+        for anc in anchors:
+            if hasattr(anc, "origin_server_certs"):
+                anchor_type = "UNKNOWN"
```

Comment:
We have SDK-1028 to change this.
Since clients _could_ potentially be using this string to switch, we have to treat it as a breaking change, even though the chance of them doing this is very minimal. We'll have to wait til v3.0.0 to change this. 
```suggestion
                anchor_type = "Unknown"
```